### DISCUSSION TITLE
CPDAILY 新增设备验证策略方案讨论

### DISCUSSION THREAD

**Discussion by yaonyan** [ceajs/cea#56] [Ideas]
see https://github.com/ceajs/cea/issues/55

**Comment by cq-s** [comment]
验证码问题经过抓包，已获得所有图片库，现在需要的是一个机器学习模型，将验证码中出现的图片分类至对应类型即可

**Reply by yaonyan** [reply]
这样成本看起来比较高，不过用于学习研究也是有价值的 👍

**Reply by yaonyan** [reply]
携带 deviceId 验证，再签带上相同 dId 吧？

**Comment by githuu5y5u** [comment]
提供一种解决方案
https://github.com/githuu5y5u/idk/blob/main/src/main/java/org/example/App.java
采用图片相似度进行检测
目前识别可以实现瞬间完成
